test/functional/hostgroups_controller_test.rb

```diff
@@ -152,6 +152,21 @@ def setup_user(operation, type = 'hostgroups')
     assert_equal(1, (assigns(:hostgroup).puppetclasses.length))
   end
 
+  test 'user with view_params rights should see parameters in a hostgroup' do
+    setup_user "edit"
+    setup_user "view", "params"
+    hg = FactoryGirl.create(:hostgroup, :with_parameter)
+    get :edit, {:id => hg.id}, set_session_user.merge(:user => users(:one).id)
+    assert_not_nil response.body['Parameter']
+  end
+
+  test 'user without view_params rights should not see parameters in a hostgroup' do
+    setup_user "edit"
+    hg = FactoryGirl.create(:hostgroup, :with_parameter)
+    get :edit, {:id => hg.id}, set_session_user.merge(:user => users(:one).id)
+    assert_nil response.body['Parameter']
```

Comment:
This test fails because the tab is still displayed (for class parameters to show). This should be changed to `assert_nil response.body['Global parameters']`